MovieHub/MovieHub/Data/Local/LocalPushNotification.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  LocalPushNotification.swift
+//  MovieHub
+//
+//  Created by Duy Nguyễn on 25/04/2024.
+//
+
+import Foundation
+import UserNotifications
+
+final class LocalPushNotification {
+    static let shared = LocalPushNotification()
+    private init() {}
+    private let notificationCenter = UNUserNotificationCenter.current()
+    
+    func checkForPermissionNotification() {
+        notificationCenter.getNotificationSettings { settings in
+            switch settings.authorizationStatus {
+            case .notDetermined:
+                self.requestNotificationPermission()
+            case .denied:
+                return
+            case .authorized:
+                self.scheduleDailyNotification()
+            default:
+                return
+            }
+        }
+    }
+    
+    private func requestNotificationPermission() {
+        notificationCenter.requestAuthorization(options: [.alert, .sound]) { didAllow, error in
```

Comment:
k handle error thì bỏ nó đi e 
```suggestion
        notificationCenter.requestAuthorization(options: [.alert, .sound]) { didAllow, _ in
```